ts/src/test/static/request/huobi.json

```diff
@@ -470,6 +484,18 @@
                     "LTC/USDT"
                 ],
                 "output": "{\"symbol\":\"ltcusdt\"}"
+            },
+            {
+                "description": "Swap cancel all trailing orders",
+                "method": "cancelAllOrders",
+                "url": "https://api.hbdm.com/linear-swap-api/v1/swap_cross_track_cancelall?AccessKeyId=bgbfh5tv3f-83da6485-1bbb64ae-3231b&SignatureMethod=HmacSHA256&SignatureVersion=2&Timestamp=2024-01-03T01%3A01%3A08&Signature=ug%2FwGTlkVE%2BmL5C%2BkBJV83nXrF8%2B8pLKOtSpdflruCU%3D",
+                "input": [
+                  "BTC/USDT:USDT",
+                  {
+                    "trailing": true
+                  }
+                ],
+                "output": "{\"contract_code\":\"BTC-USDT\"}"
```

Comment:
@Dan-krm I think that these new tests are causing Travis to fail. The errors below are happening on every PR

```
[TEST_FAILURE][PY][STATIC_REQUEST_TEST_FAILURE][huobi  spot [subType: linear] ][fetchOrders][Swap trailing orders]url mismatch expected "/linear-swap-api/v3/swap_cross_hisorders" received: "/linear-swap-api/v1/swap_cross_track_hisorders"

[TEST_FAILURE][PY][STATIC_REQUEST_TEST_FAILURE][huobi  spot [subType: linear] ][cancelAllOrders][Swap cancel all trailing orders]url mismatch expected "/linear-swap-api/v1/swap_cross_cancelall" received: "/linear-swap-api/v1/swap_cross_track_cancelall"
```

Reply:
My changes in this PR didn't transpile to the other languages, I'm not really sure what the reason for that is